tests/PHPStan/Analyser/TypeSpecifierTest.php

```diff
@@ -1133,9 +1133,7 @@ public function dataCondition(): iterable
 					new Arg(new Variable('stringOrNull')),
 					new Arg(new Expr\ConstFetch(new Name('false'))),
 				]),
-				[
-					'$object' => 'object',
```

Comment:
```
if (is_subclass_of($object, $stringOrNull, false))
```
specified `$object` as `object` but this was already known so it's not a big loss.